.changeset/brave-brooms-invent.md

```diff
@@ -0,0 +1,5 @@
+---
+"@rocket.chat/meteor": patch
+---
+
+Fixed a problem that caused visitor creation to fail when GDPR setting was enabled and visitor was created via Apps Engine or the deprecated `livechat:registerGuest` method.
```

Comment:
```suggestion
Fixes a problem that caused visitor creation to fail when GDPR setting was enabled and visitor was created via Apps Engine or the deprecated `livechat:registerGuest` method.
```
Not an issue, but I found out we changed our changeset best practices docs and we should now use the present tense. I got surprised by that, just found an opportunity to spread the word :kekw:

Reply:
Well